keda/datadog_checks/keda/metrics.py

```diff
@@ -0,0 +1,145 @@
+    "apiserver_delegated_authz_request_duration_seconds": "apiserver_delegated_authz_request_duration_seconds",
+    "apiserver_envelope_encryption_dek_cache_fill_percent": "apiserver_envelope_encryption_dek_cache_fill_percent",
+    "apiserver_flowcontrol_read_vs_write_current_requests": "apiserver_flowcontrol_read_vs_write_current_requests",
+    "apiserver_flowcontrol_seat_fair_frac": "apiserver_flowcontrol_seat_fair_frac",
+    "apiserver_request": "apiserver_request",
+    "apiserver_request_duration_seconds": "apiserver_request_duration_seconds",
+    "apiserver_request_filter_duration_seconds": "apiserver_request_filter_duration_seconds",
+    "apiserver_request_sli_duration_seconds": "apiserver_request_sli_duration_seconds",
+    "apiserver_request_slo_duration_seconds": "apiserver_request_slo_duration_seconds",
+    "apiserver_response_sizes": "apiserver_response_sizes",
+    "apiserver_storage_data_key_generation_duration_seconds": "apiserver_storage_data_key_generation_duration_seconds",
+    "apiserver_storage_data_key_generation_failures": "apiserver_storage_data_key_generation_failures",
+    "apiserver_storage_envelope_transformation_cache_misses": "apiserver_storage_envelope_transformation_cache_misses",
+    "apiserver_tls_handshake_errors": "apiserver_tls_handshake_errors",
+    "apiserver_webhooks_x509_insecure_sha1": "apiserver_webhooks_x509_insecure_sha1",
+    "apiserver_webhooks_x509_missing_san": "apiserver_webhooks_x509_missing_san",
+    "authenticated_user_requests": "authenticated_user_requests",
+    "authentication_attempts": "authentication_attempts",
+    "authentication_duration_seconds": "authentication_duration_seconds",
+    "authorization_attempts": "authorization_attempts",
+    "authorization_duration_seconds": "authorization_duration_seconds",
+    "cardinality_enforcement_unexpected_categorizations": "cardinality_enforcement_unexpected_categorizations",
+    "disabled_metrics": "disabled_metrics",
+    "field_validation_request_duration_seconds": "field_validation_request_duration_seconds",
+    "hidden_metrics": "hidden_metrics",
+    "registered_metrics": "registered_metrics",
+    # Generic Metrics
+    'controller_clientset_k8s_request': 'controller.clientset.k8s.request',
+    'go_gc_duration_seconds': 'go.gc.duration.seconds',
+    'go_goroutines': 'go.goroutines',
+    'go_info': "go.info",
+    'go_memstats_alloc_bytes': {'name': 'go.memstats.alloc_bytes', 'type': 'native_dynamic'},
+    'go_memstats_buck_hash_sys_bytes': 'go.memstats.buck_hash.sys_bytes',
+    'go_memstats_frees': 'go.memstats.frees',
+    'go_memstats_gc_cpu_fraction': 'go.memstats.gc.cpu_fraction',
+    'go_memstats_gc_sys_bytes': 'go.memstats.gc.sys_bytes',
+    'go_memstats_heap_alloc_bytes': 'go.memstats.heap.alloc_bytes',
+    'go_memstats_heap_idle_bytes': 'go.memstats.heap.idle_bytes',
+    'go_memstats_heap_inuse_bytes': 'go.memstats.heap.inuse_bytes',
+    'go_memstats_heap_objects': 'go.memstats.heap.objects',
```

Comment:
Do we want to use the `DEFAULT_GO_METRICS` constant to handle these generic go metrics?